### PR TITLE
Refactored step definitions

### PR DESCRIPTION
**Pull request by Luisjor**

### CODE REVIEW

File: 3.Tests/UI/StepDefinitions/Settings/Profile/UpdateFullNameStepDefinition.cs [hunk 1]
```diff
@@ -8,18 +8,21 @@ namespace SeleniumTest.Tests;
 [Scope(Feature = "Full Name update")]
```
Comment (gioacchinogioffre) [inline 3.Tests/UI/StepDefinitions/Settings/Profile/UpdateFullNameStepDefinition.cs:8]: I already remove this file in a new PR so you can omit these changes please.

File: 3.Tests/UI/StepDefinitions/Settings/Profile/UpdateTimeZoneStepDefinition.cs [hunk 2]
```diff
@@ -8,18 +8,23 @@ namespace SeleniumTest.Tests;
 [Scope(Feature = "Time Zone update")]
```
Comment (gioacchinogioffre) [inline 3.Tests/UI/StepDefinitions/Settings/Profile/UpdateTimeZoneStepDefinition.cs:8]: I already remove this file in a new PR so you can omit these changes please.

**Review by gioacchinogioffre — CHANGES_REQUESTED** [review]
Please lemme know if you have any doubts with my comments.

**Review by svaleria — APPROVED** [review]

File: 3.Tests/UI/StepDefinitions/Settings/Profile/UpdateTimeZoneStepDefinition.cs (hunk 2 above)
Comment (Luisjor) [reply 3.Tests/UI/StepDefinitions/Settings/Profile/UpdateTimeZoneStepDefinition.cs]: Done

File: 3.Tests/UI/StepDefinitions/Settings/Profile/UpdateFullNameStepDefinition.cs (hunk 1 above)
Comment (Luisjor) [reply 3.Tests/UI/StepDefinitions/Settings/Profile/UpdateFullNameStepDefinition.cs]: Done

**Review by gioacchinogioffre — APPROVED** [review]

**Review by sebasonti-jalauni — APPROVED** [review]
Just take care of the warnings

**Pull request merged**